src/util.h

```diff
@@ -49,6 +50,20 @@ NORETURN void Fatal(const char* msg, ...);
 #define NINJA_FALLTHROUGH
 #endif
 
+/// Whether C++11 features are enabled or not.
+#if __cplusplus >= 201103L || (defined(_MSC_VER) && _MSC_VER >= 1900)
+#  define NINJA_CPP11 true
+#else
+#  define NINJA_CPP11 false
```

Comment:
Also, technically you could change this to simply

#define NINJA_CPP11 __cplusplus >= 201103L || (defined(_MSC_VER) && _MSC_VER >= 1900)

and get the same behavior, without having to do #if / #else / #endif.